lib/src/rules/require_trailing_commas.dart

```diff
@@ -163,6 +163,15 @@ class _Visitor extends SimpleAstVisitor<void> {
       return true;
     }
 
+    // Exception is allowed if the last parameter is a anonymous function call.
```

Comment:
nit: "parameter" --> "argument".

It's wrong in a couple of previous cases as well, but fixing them is even more optional.

Reply:
Done